libs/core/src/components/pds-chip/pds-chip.scss

```diff
@@ -21,7 +21,7 @@
   --color-background-warning-dot: var(--pine-color-yellow-300);
   --color-background-warning-interactive: var(--pine-color-yellow-200);
 
-  --font-size-sm: var(--pine-font-size-087);
+  --font-size-sm: var(--pine-font-size-85);
```

Comment:
is there a reason why it's not `085`? It seems all other two digits follow the three digit placeholder sequence